client/src/app/site/pages/meetings/pages/motions/pages/motion-export/components/motion-export/motion-export.component.ts

```diff
@@ -0,0 +1,444 @@
+            this.deselectOption(`pageLayout`, `addBreaks`);
+            this.changeStateOfChipOption(this.addBreaksChip, true, `addBreaks`);
+            this.changeStateOfChipOption(this.tableOfContentChip, true, `toc`);
+        } else {
+            this.changeStateOfChipOption(this.addBreaksChip, false, `addBreaks`);
+            this.changeStateOfChipOption(this.tableOfContentChip, false, `toc`);
+        }
+    }
+
+    // Helper function to exclude option from formcontrol and deselect it
+    private deselectOption(formGroup: string, value: string): void {
+        this.dialogForm
+            .get(formGroup)
+            .setValue(this.dialogForm.get(formGroup).value?.filter(obj => !obj.includes(value)));
+    }
+
+    // Helper function to determine if mat-chip-option should be selected
+    // Needed, because using the binding with formControl does not disable mat-chip-option
+    public isDisabled(value: string): boolean {
+        return this.disabledControls.includes(value);
+    }
+
+    // Helper function to filter form control values as set as default based on the meeting settings
+    public filterFormControlDefaults(formControl: string, setting: keyof Settings, value?: string): void {
+        value = value ? value : setting.toString();
+        this.dialogForm
+            .get(formControl)
+            .setValue(
+                !this.meetingSettingsService.instant(setting)
+                    ? this.dialogForm.get(formControl).value.filter(obj => !obj.includes(value))
+                    : this.dialogForm.get(formControl).value
+            );
+    }
+
+    // Helper function to filter and disable form control values based on availability of the properties
+    // in the chosen motions
+    public filterFormControlAvailableValues(formControl: string, val: string, chipOption: MatChipOption): void {
+        if (this.motions_models.filter(m => (m[val] === null || m[val]?.length === 0 ? false : m[val])).length === 0) {
+            this.deselectOption(formControl, val);
+            this.changeStateOfChipOption(chipOption, true, val);
+        }
+    }
+
+    public async exportMotions(): Promise<void> {
+        this.repoSub.unsubscribe();
+        this.exportForm = this.formBuilder.group({
+            format: [],
+            lnMode: [],
+            crMode: [],
+            content: [],
+            metaInfo: [],
+            pdfOptions: [],
+            comments: []
+        });
+        const motions_models = this.motions.map(motion => this.motionRepo.getViewModel(motion));
+        this.exportForm.patchValue(this.motionExportDialogService.dialogToExportForm(this.dialogForm));
+        await this.motionExportDialogService.export(this.exportForm.value, motions_models);
+        // The timeout is needed for the repos to update their view model list subjects
+        setTimeout(() => {
+            this.exportService.evaluateExportRequest(this.exportForm.value, motions_models);
+        }, 2000);
```

Comment:
You are using `await this.modelRequestService.fetch` in `motionExportDialogService.export` which resolves after the view model store was updated. Normally this should work without the timeout.